netbout-client/pom.xml

```diff
@@ -101,13 +101,13 @@
         <dependency>
             <groupId>com.sun.jersey</groupId>
             <artifactId>jersey-client</artifactId>
-            <version>1.18.1</version>
+            <version>1.19</version>
             <scope>runtime</scope>
         </dependency>
         <dependency>
             <groupId>com.jcabi</groupId>
             <artifactId>jcabi-http</artifactId>
-            <version>1.10.4</version>
+            <version>1.14.1</version>
```

Comment:
@exper0 let's upgrade to latest 
https://github.com/jcabi/jcabi-http/releases/tag/1.14.2